dj_backend_server/entrypoint-nginx.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
Actually the issue is that the docker start and you cannot edit the conf file because is locked. So this is why I did not found a solution yet. If I stop the NGINX will drop me out so cannot edit. 
A solution would be to have the nginx.template.conf with a variable and replace it from makefile. and build it after ....